developer-hub/src/polkadot_sdk/templates.rs

```diff
@@ -0,0 +1,21 @@
+//! # Templates
+//!
+//! - classic [`substrate-node-template`](https://github.com/substrate-developer-hub/substrate-node-template)
+//! - new minimal-template: TODO
+//! - classic [cumulus-parachain-template](https://github.com/substrate-developer-hub/substrate-parachain-template)
+//! - [`extended-parachain-template`](https://github.com/paritytech/extended-parachain-template)
+//! - [`frontier-parachain-template`](https://github.com/paritytech/frontier-parachain-template)
```

Comment:
I think worth including a single sentence description along with each of these links

Reply:
1896ac74b52b5302671ba82f0852819945b9c531